src/components/credits/Contributor.jsx

```diff
@@ -0,0 +1,17 @@
+import ExternalLink from "@/components/shared/ExternalLink";
+
+export default function Contributor(props) {
```

Comment:
Please use props destructuring instead: https://medium.com/@lcriswell/destructuring-props-in-react-b1c295005ce0

It helps with typing and you can (should) add prop-types: https://reactjs.org/docs/typechecking-with-proptypes.html#function-components